randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -2312,7 +2304,9 @@ Extra - asset_id: collision_camera_028
   > Door to Save Station North
       After Ferenia - db_reg_b2_005
   > Event - Wave Blob (db_reg_b2_005)
-      Wave Beam
+      Any of the following:
+          Wave Beam
+          Pseudo-Wave Beam (Beginner) and Shoot Beam
```

Comment:
you didn't include the Slide here

Reply:
For me in the Data Visualizer it shows slide.. lol